packages/components/src/font-size-picker/index.js

```diff
@@ -12,18 +12,22 @@ import Button from '../button';
 import RangeControl from '../range-control';
 import SelectControl from '../select-control';
 
+const DEFAULT_FONT_SIZE = 'default';
+const CUSTOM_FONT_SIZE = 'custom';
+
 function getSelectValueFromFontSize( fontSizes, value ) {
 	if ( value ) {
 		const fontSizeValue = fontSizes.find( ( font ) => font.size === value );
-		return fontSizeValue ? fontSizeValue.slug : 'custom';
+		return fontSizeValue ? fontSizeValue.slug : CUSTOM_FONT_SIZE;
 	}
-	return 'normal';
+	return DEFAULT_FONT_SIZE;
 }
 
 function getSelectOptions( optionsArray ) {
 	return [
+		{ value: DEFAULT_FONT_SIZE, label: __( 'Default' ) },
 		...optionsArray.map( ( option ) => ( { value: option.slug, label: option.name } ) ),
-		{ value: 'custom', label: __( 'Custom' ) },
+		{ value: CUSTOM_FONT_SIZE, label: __( 'Custom' ) },
```

Comment:
If I select a font size in the select control and then I select the custom options, I get some JavaScript errors:
```
react-dom.8b3dda97.js?ver=16.9.0:481 Uncaught TypeError: Cannot read property 'size' of undefined
    at onSelectChangeValue (index.js:65)
    at onChangeValue (index.js:35)
    at HTMLUnknownElement.callCallback (react-dom.8b3dda97.js?ver=16.9.0:341)
    at Object.invokeGuardedCallbackDev (react-dom.8b3dda97.js?ver=16.9.0:391)
    at invokeGuardedCallback (react-dom.8b3dda97.js?ver=16.9.0:448)
    at invokeGuardedCallbackAndCatchFirstError (react-dom.8b3dda97.js?ver=16.9.0:462)
    at executeDispatch (react-dom.8b3dda97.js?ver=16.9.0:594)
    at executeDispatchesInOrder (react-dom.8b3dda97.js?ver=16.9.0:616)
    at executeDispatchesAndRelease (react-dom.8b3dda97.js?ver=16.9.0:719)
    at executeDispatchesAndReleaseTopLevel (react-dom.8b3dda97.js?ver=16.9.0:727)
```

Reply:
It should have been fixed on c27a7409b47bd370c229c4c5bef0eec7c8050d7c